coalib/coala.py

```diff
@@ -61,13 +105,75 @@ def main():
 
             return 0
 
+        elif args.non_interactive:
+            import functools
+            from coalib.output.ConsoleInteraction import print_results_no_input
+
+            partial_print_sec_beg = functools.partial(
+                print_section_beginning,
+                console_printer)
+            results, exitcode, _ = run_coala(
+                print_results=print_results_no_input,
+                print_section_beginning=partial_print_sec_beg,
+                force_show_patch=True,
+                console_printer=console_printer)
+
+            return exitcode
+
+        elif args.format:
+            from coalib.output.ConsoleInteraction import \
+                    print_results_formatted
+
+            results, exitcode, _ = run_coala(
+                    print_results=print_results_formatted)
+
+            return exitcode
+
+        elif args.clean:
```

Comment:
IMO the cleaning should still be in an own binary, it performs a distinct job. The others are doing the same thing, just the output change, right?